olm-catalog/serverless-operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.5"
+LABEL operators.operatorframework.io.bundle.channels.v1="4.5"
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-serverless-rhel8-operator-container" \
+      name="openshift-serverless-1/serverless-rhel8-operator" \
+      version="1.8.0" \
+      summary="Red Hat OpenShift Serverless 1 Serverless Operator" \
```

Comment:
This summary could probably be more user friendly. I don't know if it makes sense to mantion "Serverless Operator" twice and the `1`. Similarly for the `description` and `display-name`. That's probably a question for @warrenvw why they have this format.

Reply:
Yeah, deferring this to @warrenvw. I dunno where this is even used at all.

Reply:
This was setup in part before I joined the team. I kept the same format for all the repos I requested. In the interest of merging this PR, let's leave as is. If necessary, we can fix in a future PR.